src/main/kotlin/pt/up/fe/ni/website/backend/controller/ProjectController.kt

```diff
@@ -0,0 +1,28 @@
+package pt.up.fe.ni.website.backend.controller
+
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import pt.up.fe.ni.website.backend.model.Project
+import pt.up.fe.ni.website.backend.service.ProjectService
+
+@RestController
+@RequestMapping("/projects")
+class ProjectController(private val service: ProjectService) {
+
+    @GetMapping("/")
+    fun getAllProjects() = service.getAllProjects()
+
+    @PostMapping("/new")
+    fun createNewProject(@RequestBody project: Project) = service.createProject(project)
```

Comment:
https://rules.sonarsource.com/java/tag/spring/RSPEC-4684

Reply:
Found this online

Reply:
Thank you! We should open issues to fix these vulnerabilities!

Reply:
That is a very nice point. Maybe we should implement the proper DTO pattern in our project. It felt over-engineered to me but maybe it's worth it

Reply:
I'm almost finishing my DTO implementation. The only part that is missing is saving the result to the DB. I'll push it once it's ready!

Reply:
I've installed SonarLint on IntellIJ and it didn't detect this vulnerability because we are using Kotlin :upside_down_face: 
Looks like we'll just have to be very careful...